code/style.css

```diff
@@ -1,4 +1,285 @@
+/* 
+This CSS file is created with mobile first in mind.
+Desktop and Tablet is added at the bottom under media sections.
+Author Jenny Quach
+*/
+
```

Comment:
Nice text with good explanation. And I liked that you wrote Author. I will take that with me and do the same ;)

Reply:
Thanks :)